samples/bluetooth/peripheral_lbs/README.rst

```diff
@@ -6,7 +6,7 @@ Bluetooth: Peripheral LBS
 Overview
 ********
 
-This sample demonstrates LBS (LED Button) GATT Service. When connected
+This sample demonstrates how to use the :ref:`lbs_readme`. When connected
```

Comment:
I know it is not part of this PR, but Readme for this sample does not conform to the sample docs skeleton (lacks Testing Steps, interface description etc..).

We could either extend the sample description in this PR or in the future one.

Reply:
If we can get it done in this PR, that would be nice. :) But the "bad" docs are in already, so it's not a hard requirement.
How much of the content can just be copied from the nRF5 SDK?

Reply:
I think that LED signalling is a bit different in NCS, so it's probably a better idea if either I or @KAGA164 proposes a draft with the docs extension. While we at it, we will have a chance to double check if other things in testing steps are correct.

Yes, all of this can be done in the subsequent PR :)